content/TrueCommand/TCGettingStarted/Install/_index.md

```diff
@@ -115,23 +110,81 @@ Click the **Discovered Systems** icon and select the TrueNAS system. TrueCommand
 It can take 10 to 15 minutes for the TrueNAS instance to fully sync up with TrueCommand Cloud.
 When all systems are connected to TrueCommand Cloud, refer to the [TrueCommand Administration articles]({{< relref "/TrueCommand/Administration/_index.md" >}}) for more instructions about setting up configuration backups, alerts, reports, and role-based access control.
 {{< /tab >}}
+{{< tab "VM Deployment" >}}
+
+Deploying TrueCommand on a virtual machine (VM) requires different methods depending on which operating system you intend to use. 
+
+You can find VM images and setup instructions on our [TrueCommand-install](https://github.com/ixsystems/truecommand-install) GitHub repository.
+
+## Linux
+
+### Debian
+
+If you don't already have it, download the VM image [here](https://www.debian.org/CD/http-ftp/).
+
+{{< hint info >}}
+Ensure you have the "wget" utility installed first: `apt-get install wget`
+{{< /hint >}}
+
+Run this command (as root) from a system terminal:  
+`wget https://raw.githubusercontent.com/iXsystems/truecommand-install/main/debian/setup.sh -O - | bash`
+
+### Alpine
+
+If you don't already have it, download the VM image [here](https://alpinelinux.org/downloads/).
+
+{{< hint info >}}
+Ensure you have the "wget" utility installed first: `apk add wget`.   
+Ensure that you have the "community" package repository enabled:
```

Comment:
first instance of a new term use italics and define or explain it, don't use double quotes
```suggestion
Ensure that you have the community package repository enabled:
```